.config/ansible-lint.yml

```diff
@@ -6,3 +6,12 @@
 exclude_paths:
   - .cache/  # implicit unless exclude_paths is defined in config
   - .yamllint
+  - ../molecule/
+  - ../.github/
```

Comment:
why the ".."?
and why exclude the molecule folder, you fixed a lot of stuff there :)